packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -438,7 +438,7 @@ end
         bitcode: true,
         reportTimings: false,
         iosBuildArchs: <DarwinArch>[DarwinArch.armv7, DarwinArch.arm64],
-        dartDefines: dartDefines,
+        dartDefines: stringsArg('dart-define'),
```

Comment:
Why not take `BuildInfo` and extract dart defines out of it?

Reply:
Done